rankwatch-projects/src/auth.js

```diff
@@ -0,0 +1,63 @@
+var cc = DataStudioApp.createCommunityConnector();
+var USERNAME_PROPERTY_PATH = 'dscc.username';
+var PASSWORD_PROPERTY_PATH = 'dscc.password';
+
+// TODO - implement your credentials validation logic here.
+function validateCredentials(username, password) {
+  var rawResponse = UrlFetchApp.fetch(
+    'https://apiv2dev.rankwatch.com/user/profile/json/',
+    {
+      method: 'GET',
+      headers: {
+        Authorization:
+          'Basic ' + Utilities.base64Encode(username + ':' + password)
+      }
+    }
+  );
+  content = JSON.parse(rawResponse);
+  return content['errno'] == 0;
+}
+
+// https://developers.google.com/datastudio/connector/auth#getauthtype
+function getAuthType() {
+  return cc
+    .newAuthTypeResponse()
+    .setAuthType(cc.AuthType.USER_PASS)
+    .setHelpUrl('https://www.example.org/connector-auth-help')
```

Comment:
This should either be removed, or the url replaced with a page from rankwatch that helps explain how to get the necessary username & password.

Reply:
This has been removed.